app/policies/feed_token_policy.rb

```diff
@@ -0,0 +1,21 @@
+class FeedTokenPolicy < ApplicationPolicy
+  def show?
+    update?
```

Comment:
Same as with the `Feed` ... I think readonly users should be able to see these.  (But the FormBuilder will disable the fields, so you can't edit them).